ddio_lnx/lnxio.cpp

```diff
@@ -99,3 +99,11 @@ void ddio_DebugMessage(unsigned err, char *fmt, ...) {
   mprintf((0, "%s\n", buf));
 }
 
+char* getExecutablePath(char *exec_path, size_t len) {
+  if (realpath("/proc/self/exe", exec_path) == NULL) {
```

Comment:
this won't work on macos. it needs to be a call to `_NSGetExecutablePath`

Reply:
Hopefully I did that right. I'm not familiar with mac filesystem stuff.

Reply:
looks and works good